tests/performance/feeding_and_recovery/testplot.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+lastdir=$(ls /home/y/logs/systemtests/command-line/ | sort | tail -1)
```

Comment:
Probably not used ? Reference to /home/y.

Reply:
Good point, removed